### ISSUE TITLE
Failure when running "ekco-purge-node --help"

### ISSUE THREAD

**Issue by yasserg** [replicatedhq/kURL#901]
```
ekco-purge-node --help
error: Internal error occurred: error executing command in container: failed to exec in container: failed to start exec "95c0981dec8d1d66ae48406ab9f2efdea91ae34e85ea3bce8d8e9678a1dc10f1": OCI runtime exec failed: exec failed: container_linux.go:367: starting container process caused: exec: "purge-node": executable file not found in $PATH: unknown
```

Environment:

```
kubernetes:
    version: 1.19.3
  weave:
    version: 2.6.5
  rook:
    version: 1.4.3
    isBlockStorageEnabled: true
    blockDeviceFilter: sdb1|nvme1n1
  registry:
    version: 2.7.1
  kotsadm:
    version: 1.21.2
    applicationSlug: forward-cluster/beta
  ekco:
    version: 0.6.0
  containerd:
    version: 1.3.7
```

Installed on CentOs 8.2

**Comment by yasserg** [comment]
Note that executing the command from inside pod works:

```
kubectl exec -n kurl -it ekc-operator-5bc9c98b96-npcfm -- /bin/bash
root@ekc-operator-5bc9c98b96-npcfm:/# ekco purge-node n3
2020-11-13T05:27:23.059Z	INFO	cluster/purge_node.go:17	Purge node "n3"
2020-11-13T05:27:23.148Z	INFO	cluster/rook_ceph.go:66	Removing node "n3" from CephCluster storage list
2020-11-13T05:27:23.161Z	INFO	cluster/rook_ceph.go:77	Purge node "n3": removed from CephCluster node storage list
2020-11-13T05:27:23.274Z	INFO	cluster/rook_ceph.go:108	Deleted OSD Deployment for node n3
2020-11-13T05:27:25.482Z	INFO	cluster/purge_node.go:42	Purge node "n3": ceph osd purge command executed
2020-11-13T05:27:25.487Z	INFO	cluster/purge_node.go:89	Purge node "n3": deleted Kubernetes Node object
```

**Comment by franklines** [comment]
@yasserg, I also ran into this issue. I filed the PR (https://github.com/replicatedhq/kURL/pull/876) to address it.

**Comment by areed** [comment]
Fixed by #876
